src/main/java/seedu/address/logic/commands/ShowNotesCommand.java

```diff
@@ -27,12 +27,12 @@
 
     private static final String LINE_SEPARATOR = "====================================================================";
 
-    private static final String MESSAGE_FILE_LOCATION_ERROR = "The folder \"notes\" is not found!\r\n"
+    private static final String MESSAGE_FILE_LOCATION_ERROR = "The folder \"notes\" is not found!" + NEWLINE_SEPARATOR
```

Comment:
Wait.  I thought that we agreed that our implementation causes the notes folder to be created the moment NSync is initialised?